js/apps/events-page/partials/eventDetails.html

```diff
@@ -27,16 +27,27 @@ <h1>{{eventDetailsPage.data.Title}}</h1>
 
 		<p ng-if="eventDetailsPage.data.Description">{{eventDetailsPage.data.Description}}</p>
 
-		<p class="registration-required" ng-show="eventDetailsPage.isRegistrationRequired && !eventDetailsPage.isOver">
-			<a href="#!/register/{{eventDetailsPage.data.EventId}}" class="btn-primary btn-uppercase-bold" ng-if="eventDetailsPage.data.RegistrationTypeCodeEnum === 1">
+		<p ng-if="eventDetailsPage.data.isFull">
+			<strong>
+				This event is currently full.
+				<span ng-if="eventDetailsPage.data.isFull && eventDetailsPage.data.isWaiting">You may still register for the waiting list.</span>
```

Comment:
I'm not sure how much of a pain it would be, but it seems like data could be event here.

eventDetailsPage.event.isFull.  Not a killer but would make it a little cleaner